config/galaxy.ini.sample

```diff
@@ -1256,6 +1256,11 @@ use_interactive = True
 # done in memory, which is a bit quicker.
 #track_jobs_in_database = True
 
+# Usually, a job is only killed if all output datasets are deleted in the history.
+# This option let's galaxy kill the job after only one of the jobs output datasets
+# is deleted
+#delete_output_dataset_kills_job = True
```

Comment:
The value in the commented option should be the default, which below is correctly defined as `False`.